wp-content/themes/missilethreat/assets/_scss/layout/_footer.scss

```
@@ -1 +1,103 @@
 @use 'abstracts' as *;
+
+.footer {
+  @extend %font-body-short-md;
+  @include full-width-background($color-primary-blue-100);
+  padding-top: rem(40px);
+  padding-bottom: rem(16px);
+  color: $color-text-light-170;
+  background-color: $color-primary-blue-100;
+}
+
+.footer__contact {
+  display: flex;
+  flex-wrap: nowrap;
+  margin-top: rem(32px);
+}
+
+.footer_address {
+  margin-left: rem(32px)
+}
+
+.footer__logo, .footer_mdp, .footer_pp {
+  color: $color-text-light-100;
+}
+
+.footer_nav_side {
+  color: $color-text-light-190;
+}
+
+.footer__copyright {
+  color: $color-text-light-155;
+}
+
+.footer__logo {
+  margin-bottom: rem(24px);
+}
+
+.footer_social_icons {
+  display: flex;
+  align-items: center;
+}
+
+.facebook_icon, .twitter_icon {
+  padding: rem(8px) rem(9px);
+  border-radius: rem(50px);
+  background-color: $color-text-light-100;
+  color: $color-primary-blue-100;
+  vertical-align: middle;
+}
+
+.twitter_icon {
+  margin-left: rem(8px)
+}
+
+.footer__contact {
+  margin-bottom: rem(32px);
+}
+
+.footer_news_letter {
+  margin-bottom: rem(32px);
+}
+
+.placeholder_news_nav {
+  margin-bottom: rem(48px);
+}
+
+@include breakpoint('medium') {
+
+  .placeholder_news_nav {
+    display: flex;
+    flex-wrap: nowrap;
+  }
+
+  .footer_news_letter {
+    max-width: rem(350px);
+  }
+
+  .footer_nav_side {
+    margin-left: rem(48px);
+    max-width: rem(165px);
+  }
+
+} 
+
+@include breakpoint('large') {
+  .footer_large_container{
+    display: flex;
+    flex-wrap: nowrap;
```

Let's make this `display:grid;` and remove the `max-width` on the items in the container. 
```suggestion
  .footer_large_container{
    display: grid;
    grid-template-columns: 1fr 1fr;
    margin-bottom: rem(32px);
  }

  .placeholder_news_nav {
    margin-left: rem(48px);
  }
  
  .footer__contact {
    margin-bottom: 0;
  }
```